tests/test_main.py

```diff
@@ -81,6 +82,49 @@ def test_drop_data(
         assert mocked_sink.call_args == call(expected_content, expected_file)
 
 
+@pytest.mark.parametrize(
+    "api_key, x_system, expected_response_code, files",
+    [
+        ("api-test-key", "test_system", 202, {"file1.txt": "file1 content"}),
```

Comment:
```suggestion
        ("api-test-key", "test_system", 202, {"file1.txt": "file1 content", "file2.csv": "1,2,3"}),
```

let's try with a second file as well, to justify the test name `test_drop_multiple_files` 🙃